src/perso.ml

```diff
@@ -113,6 +113,10 @@ value string_of_title conf base and_txt p (nth, name, title, places, dates) =
                    else Buffer.add_string b "-"
                | _ -> () ];
                match date_end with
+               [ Some (Dtext d ) ->
```

Comment:
I think this is wrong. Looking at the code now, it's already done like 2 lines above